apps/model-diagnostics/model_diagnostics.cpp

```diff
@@ -0,0 +1,52 @@
+/*************************************************
```

Comment:
would be nice to also have a Python script for that, not just .cpp app.

Reply:
Why is it important? It's debug tool for DNN networks. Python bindings adds extra abstraction layer and make debugging harder.